Pizzeria.ServiceDefaults/Pizzeria.ServiceDefaults.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>

```

Comment:
These 3 properites are already defined in `Directory.Build.props` and can be removed